src/storm/solver/helper/ValueIterationOperator.cpp

```diff
@@ -44,13 +46,33 @@ void ValueIterationOperator<ValueType, TrivialRowGrouping, SolutionType>::setMat
             matrixColumns.back() = StartOfRowGroupIndicator;  // This is the start of the next row group
         }
     } else {
-        matrixColumns.push_back(StartOfRowIndicator);  // Indicate start of first row
-        for (auto rowIndex : indexRange<Backward>(0, numRows)) {
-            for (auto const& entry : matrix.getRow(rowIndex)) {
-                matrixValues.push_back(entry.getValue());
-                matrixColumns.push_back(entry.getColumn());
+        if constexpr (std::is_same<ValueType, storm::Interval>::value) {
+            applyCache.hasOnlyConstants.clear();
+            applyCache.hasOnlyConstants.grow(matrix.getRowCount());
+            // TODO Implement hasTwoSuccessors
```

Comment:
TODO?

Reply:
I'll just remove the `hasTwoSuccessors` thing for now :)